uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/entitlement-model_test.js

```diff
@@ -0,0 +1,86 @@
+define([
+  'leave-absences/shared/apis/entitlement-api',
+  'leave-absences/shared/models/entitlement-model',
+  'leave-absences/shared/models/instances/entitlement-instance',
+  'mocks/apis/entitlement-api-mock',
+], function () {
+  'use strict'
+
+  describe('Entitlement', function () {
+    var $provide, Entitlement, EntitlementInstance, EntitlementAPI, $rootScope;
+
+    beforeEach(module('leave-absences.models', 'leave-absences.mocks', function (_$provide_) {
+      $provide = _$provide_;
+    }));
+
+    beforeEach(inject(function (_EntitlementAPIMock_) {
+      $provide.value('EntitlementAPI', _EntitlementAPIMock_);
+    }));
+
+    beforeEach(inject(function (_Entitlement_, _EntitlementInstance_, _EntitlementAPI_, _$rootScope_) {
+      Entitlement = _Entitlement_;
+      EntitlementInstance = _EntitlementInstance_;
+      EntitlementAPI = _EntitlementAPI_;
+      $rootScope = _$rootScope_;
+
+      spyOn(EntitlementAPI, 'all').and.callThrough();
+      spyOn(EntitlementAPI, 'breakdown').and.callThrough();
```

Comment:
why are you spying on these methods? You are not doing anything it seems with that?